go.mod

```diff
@@ -3,9 +3,9 @@ module golang.zx2c4.com/wireguard
 go 1.19
 
 require (
-	golang.org/x/crypto v0.0.0-20220315160706-3147a52a75dd
-	golang.org/x/net v0.0.0-20220225172249-27dd8689420f
-	golang.org/x/sys v0.0.0-20220315194320-039c03cc5b86
+	golang.org/x/crypto v0.3.0
+	golang.org/x/net v0.2.0
+	golang.org/x/sys v0.2.0
```

Comment:
Oooolala when did the /x/ packages start getting versioned? Neato.

Reply:
Right!